docs/general/testing/web/index.md

```diff
@@ -0,0 +1,25 @@
+---
+uid: testing-web-clients
+title: Testing Jellyfin Web Clients
+---
+
+## Weekly Unstable Builds
+
+Weekly unstable builds ship Jellyfin Server and Jellyfin Web in a complete package. Please refer to [Testing Jellyfin Server](/docs/general/testing/server/) for more info.
+
+## Testing PRs from Cloudflare Pages Deployments
```

Comment:
This method will not work for every PR. If you don't see your changes in the deployed page you need to build it locally.

Reply:
You mean if the user modified the PR afterwards, right? Because every PR has its own up-to-date deployment unless CI is down unexpectedly (which is something we shouldn't consider for these layman's terms docs).

Reply:
We have already seen this multiple times on the web PRs where it only works when build locally I'm not sure the actual reasons because I don't know how that is deployed but that is a thing

Reply:
This is something that we should fix in our side then, nothing we should ever make users aware of.

Reply:
We need to let them know before we fix that though.